distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -58,60 +58,59 @@ type Filter_Condition
          This ensures that different ways of expressing the same character in
          the underlying binary representation are considered equal.
     @locale Locale.default_widget
-    Equal_Ignore_Case (to = (Missing_Argument.throw "to")) (locale:Locale=Locale.default) action:Filter_Action=Filter_Action.Keep
+    Equal_Ignore_Case (to = (Missing_Argument.throw "to")) (locale:Locale=Locale.default) keep_or_remove:Filter_Action=..Keep
 
     ## Does the value start with a prefix (Text only)?
 
        ? Table Operations
          It accepts a Text value to check if the value contains it. In case of
          Table operations, it can accept another column - then the corresponding
          values from the source column and the provided column are checked.
-    Starts_With (prefix = (Missing_Argument.throw "prefix")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) action:Filter_Action=Filter_Action.Keep
+    Starts_With (prefix = (Missing_Argument.throw "prefix")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) keep_or_remove:Filter_Action=..Keep
 
     ## Does the value end with a suffix (Text only)?
 
        ? Table Operations
          It accepts a Text value to check if the value contains it. In case of
          Table operations, it can accept another column - then the corresponding
          values from the source column and the provided column are checked.
-    Ends_With (suffix = (Missing_Argument.throw "suffix")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) action:Filter_Action=Filter_Action.Keep
+    Ends_With (suffix = (Missing_Argument.throw "suffix")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) keep_or_remove:Filter_Action=..Keep
 
     ## Does the value contain the substring (Text only)?
 
        ? Table Operations
          It accepts a Text value to check if the value contains it. In case of
          Table operations, it can accept another column - then the corresponding
          values from the source column and the provided column are checked.
-    Contains (substring = (Missing_Argument.throw "substring")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) action:Filter_Action=Filter_Action.Keep
+    Contains (substring = (Missing_Argument.throw "substring")) (case_sensitivity:Case_Sensitivity=Case_Sensitivity.Default) keep_or_remove:Filter_Action=..Keep
 
     ## Is equal to Nothing?
-    Is_Nothing action:Filter_Action=Filter_Action.Keep
+    Is_Nothing
 
     ## Is not equal to Nothing?
-    Not_Nothing action:Filter_Action=Filter_Action.Keep
+    Not_Nothing
 
     ## Is the value a NaN (Number only)?
-    Is_Nan action:Filter_Action=Filter_Action.Keep
+    Is_Nan
```

Comment:
This filter does not have a `Not_Nan` counterpart - thus removing `action` makes it impossible to remove NaN values, keeping non-NaN. Should we add `Not_Nan`? Tbh it feels more useful to remove NaN values than to keep only them.

I guess `Is_Finite` serves similar purpose but it has slightly different semantics than `Not_Nan` (it will remove infinities as well which may sometimes be desirable and sometimes not).